compechem/calculators/tools.py

```diff
@@ -0,0 +1,73 @@
+import os
+from rdkit import Chem
+
+
+def generate_inchikey(molfile):
+    mol = Chem.MolFromMolFile(molfile)
+    inchikey = Chem.MolToInchiKey(mol)
+    return inchikey
+
+
+def info(mol):
+    print(f"\nMolecule: {mol.name}")
+    print(f"\tNumber of atoms: {mol.atomcount}")
+    print(f"\tCharge: {mol.charge}")
+    print(f"\tSpin: {mol.spin}")
+    print("\nEnergies:")
+    print(f"\t Total: {mol.energies['total']} Eh")
+    print(f"\t Vibronic: {mol.energies['vibronic']} Eh")
+    print(f"\t Electronic: {mol.energies['electronic']} Eh")
+    print("\nCoordinates (Angstrom):")
+    for line in mol.geometry:
+        print(line, end="")
+    print("\n")
```

Comment:
Ottima idea, la implementerei come metodi `__str__` e `__repr__` dell'oggetto `Molecule`

Reply:
Studio cosa sono questi metodi e poi li implemento direttamente in `Molecule`!